MathLib/LinAlg/Dense/DenseTools.cpp

```diff
@@ -17,8 +17,8 @@
 namespace MathLib
 {
 
-void applyKnownSolution(DenseMatrix<double> &eqsA, DenseVector<double> &eqsRHS, std::size_t k,
-		double val)
+void applyKnownSolution(DenseMatrix<double> &eqsA, DenseVector<double> &eqsRHS,
+		DenseMatrixIndexType k, double val)
```

Comment:
Strictly speaking, this function deals with type `DenseMatrix`, so the index type should be `DenseMatrix<double>::IdxType` and not a typedef from `GlobalDenseMatrix`.